config/routes.rb

```diff
@@ -3,5 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-  resources :admin, only: [:index]
+  get "/admin", to: "admin#index"
```

Comment:
really curious to figure out if there's a better way here... but glad this worked for us...